conntrollers/controller.js

```diff
@@ -0,0 +1,11 @@
+class Controller {
+
```

Comment:
I reckon that these empty lines (2, 8) are unnecessary. It's a good practice to separate properties/methods with an empty line, but not at the beginning and at the end